src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -13,8 +15,11 @@ interface Props {
 export class Artwork extends React.Component<Props> {
   render() {
     return (
-      <View style={{ backgroundColor: "red" }}>
-        <Text>{this.props.artwork.title}</Text>
+      <View>
+        <Flex width="100%" style={{ backgroundColor: "gray" }} height={340} />
```

Comment:
`bg='black60'`? (`bg` prop should propgate through Flex)

Reply:
Oh, yeah. This is completely for dummy-purposes so just did the quickest thing.